components/net/fetch/cors_cache.rs

```diff
@@ -74,51 +73,20 @@ pub struct CacheRequestDetails {
     pub credentials: bool
```

Comment:
I'm wondering whether we even need this `CacheRequestDetails` struct at all... it feels like we can just do away this altogether and use `Request` instead. I think I'll file a follow-up issue to remove this.